cudax/include/cuda/experimental/__memory_resource/device_memory_pool.cuh

```diff
@@ -8,8 +8,8 @@
 //
 //===----------------------------------------------------------------------===//
 
-#ifndef _CUDAX__MEMORY_RESOURCE_CUDA_MEMORY_POOL
```

Comment:
While we are moving those, do we want to just move everything into cudax?

Reply:
I think we do, for now I would say it's a low priority item. But if we end up with vector needing `allocate_async/deallocate_async`, that would bump up the priority